lib/ssh-keys.nix

```diff
@@ -47,6 +48,7 @@
     david          = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCR931i3C3i8IS6xE/gpSx3RvtLsU0div8G69+KP4tYSyd7X3e73EL8dWfyHJWOVRgorHbDDOOC5qMcNB9vpen9hibRtGEeKUTBypm4vVqhBrPp/lPWU5aqYlulA6HKx5bwcg4Xi1kQofuJz9yQdaqvqTuWbJOibNmAUREGKqpERKNU1vWMY8c9u7EmDm2uKyoFLaLxd6r+w6bBqBy/Q0q8CztGqyi/hcQfznnhT/j0dFu5MwGyQ0z/Ihl58QhUc4NlD9SRlLPa4sBe6UxvB00LhyYP6BveFKUeJyahs4pSvgHis0gD3FbmtxQbRjUqkVcFkv5kj8DlKiGPQWaeVBJx" ];
     dorin          = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDPp8aWDUAFyDW1Xn9rh9J55rDgUK1JSIhiWw8uAyeE/xqz9Q6hLkLazoi6fKxs7Z8uRDxEN1fPKtIXl8ytu+nkxAXtIjvPqqC1/j077TejAekTZZZrbTMXaMvnf8sSMo3EZ0jszUreFajZ0LWr2UyLCR/Bbypfm6N59IH/7ENPLx2b36kLa0QJLUVlVzD3F7d96+vKB5rL9kanXY/Qp/M6jXwaBM69WlbK9bOH4+7RnVboU5gtLhH4XIq9E5e7I6Uq9Ukk0INpa8XpdNXNILMrU9LxvvIbDTaFuM7cNjaSjZtPLgptVklcT6NGbOaw2Bzx071N7AS2JN1FCrODAFWJ dorin@lenovo"];
     robcohen       = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCQkixVdLYB664tY3foVWa1iCIamHTB8E1kgRuNijMLY9h9+S7g8pZmcfxwwl3I0/xDVEKBb3wfhZLRJ2SpbhPuBaoODiYY14+ZpTwJszq0+AustGqiaUxS7HtOF6oTcVtxSbRHBC4zS1ibK0XsOTJAaD713RlunE1TSGZGkhgHF1xUNGE8DWYrPnS0DGx3jJkVqAMXSPVyyJhM+DdUrfSLxqiRuqWKJMtoZ/kHEcA+EvSp34x+EhHuljHzIbqnYbw87vO/WNgDzxmpGWtnacvffnVR1OWjVi+G8+hgILyGJBSvRQbmkhwR3SFUlPr6rzONXTW/opAj8gn3ZbJRIdGLEJmd6XuPEGG5OkdIWv0Jlu0ZRFHWj6PNxGXYGCaS06D1rKv+3aLREzO106i1odYl2EnIcWU5LSL28csTk2pJs8xpH+M8HGeeTXxoxFUcQOYVGwHPh4RUe9pUc0hWayo2uajZ+Okms34uavps2yLTzXyE3E76T+al4W1nyOG9HruAHumvDlWbe9oJdJ4mWBOvJModR6Vt8Gz2XFOf5CDcvWUlc14Qk80mC61eV3hRZi8CT12AtcQjE0bG/Tv+LkDNbNyUxU9LsSkPPcADvzp6vXOEr+uMQxXbawUwCtOgPhKN3pmoqRQ2YAfxrESXYFZ6YvEVYjzSLawbDEM/BlG+Nw== Minx"];
+    alanv          = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCvQkw2IYLIOiQysBJn5UuHkd7mgNkY0Q/TPP2uCPUGBZZGSIop4BzCnnPOYKn+zaMt4bLmdUuWHDs7+XKymV5RlmeMrptwIAL9XWX3d1AnWMCRgaE9ByXkEIMvMeKxUq3y2pY0e9zZLWsJvtZNzeTy227NQYYOxgx/IXdCacKdPEKENWBq6sPehP+9q+0uKW6zVUXHsCUhBXlE/FCwdh1HKoOSKyFYsvu8tIRF5u0crHQ1zDANw9ASRhmYoxvHgfEBY3GiwywTh/JHRuMS5CyD4PeKSOmYq6BALZVyMun0Ol8+XcOh63ljOlfZJGR/xWDaE0lFSuwlkG2UmGI7jLIJ alverbner@gmail.com" ];
```

Comment:
should probably have a separate group in subrepos utilizing this for those specific developers.

Reply:
makes sense.